src/ui/ApplicationSearch/SearchResultListItem.tsx

```diff
@@ -115,6 +151,80 @@ const SearchResultListItem = ({
 }) => {
   const [isOpen, setOpen] = useState(false);
   const [selected, setSelected] = useState<State.SearchItem>();
+  const themeContext = useContext(ThemeContext);
+  const status = useStatus();
+  const [itSystemStatus, setItSystemStatus] = useState<{
+    details: any | undefined;
+    timeChecked: Date | null;
```

Comment:
I wonder if both of these can just become optional (rather than one undefined and the other explicitly null)?

```
 const [itSystemStatus, setItSystemStatus] = useState<{
    details?: any;
    timeChecked?: Date;}>({
    details: undefined,
    timeChecked: undefined,
  });
```